lib/info.js

```diff
@@ -37,6 +37,7 @@ module.exports = function getInfo(link, options, callback) {
 
   var myrequest = options.request || request;
   var id = util.getVideoID(link);
+  var relatedVideos = null;
```

Comment:
Still don't think this should be placed here, as it only makes sense where it's retrieved from.